docs/source/starter/introduction.rst

```diff
@@ -615,7 +615,7 @@ Before installing anything, use the following templates to try it out live:
         .. raw:: html
 
             <div style='width:150px;height:auto'>
-                <a href="https://colab.research.google.com/drive/1rHBxrtopwtF8iLpmC_e7yl3TeDGrseJL?usp=sharing>">
+                <a href="https://colab.research.google.com/github/PyTorchLightning/pytorch-lightning/blob/master/pl_examples/basic_examples/rapid_templates/Rapid_prototyping_general_1.ipynb>">
```

Comment:
Why not add it to lightning tutorials?

Reply:
I thought it doesn't have anything much to provide, just some random dataset with simple code, so tutorials might not be worth it.

Reply:
btw, why is there the `_1` in the name?

Reply:
Don't know. I'll update it with something better.